contents/handbook/people/hiring-process.md

```diff
@@ -186,6 +186,14 @@ We also welcome external referrals, e.g. from:
 
 As a thank you, we will give you $50 credit for our [merch shop](https://merch.posthog.com/).
 
+## Visa sponsorship
+
+Building a diverse team is at the heart of our culture at PostHog and we are proud of be hiring internationally. In some cases, this included the need for visa sponsorship. We are currently able to provide visas in the US and the UK, under the following conditions. 
+
+- If you are already in the country on a visa (e.g. employed, youth mobility), or require a new visa to remain in the country (e.g. student converting to employed), we will cover the costs for any employee, new or current.
+- If you which to relocate and need a visa, we unfortunately will not cover the cost.
```

Comment:
This isn't a condition so shouldn't be in this list I think?

Reply:
Let me re-phrase the conditions bit then

Reply:
Typo "we are proud *to* be"